webapp/channels/src/components/admin_console/permission_schemes_settings/permission_team_scheme_settings/permission_team_scheme_settings.tsx

```diff
@@ -82,7 +82,7 @@ type State = {
     schemeDescription: string | undefined;
 };
 
-export default class PermissionTeamSchemeSettings extends React.PureComponent<Props & RouteComponentProps, State> {
+export class PermissionTeamSchemeSettings extends React.PureComponent<Props & RouteComponentProps, State> {
```

Comment:
I understand we export this to reduce the number of changes in the test. But I am not a big fan, since it is prone to import errors in the future. Any chance we can stop exporting this and just handle the test accordingly?

Reply:
I read different threads about how people deal with `shallow` not being able to work with HOC wrapped components. And it looked like there is no perfect solution available. Most people just export the named components for tests only, and that's what I did too. Some people tried using `render` instead of `shallow`, but that would make the tests "harder" by consuming more resources. So I just went the easier way. 
What kind of errors are you anticipating in the future?

Reply:
Thanks for the explanation. We can leave it like that.

My main concern is when importing, people mistakenly do `import {PermissionTeamSchemeSettings} from ...` instead of `import PermissionTeamSchemeSettings from ...`.

Reply:
That's a valid concern @larkox. I would recommend rewriting this component to functional and use hooks. Then you wouldn't need to wrap it with HOC, but just use the `useIntl` hook to get the work done.